plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/app/DBPProject.java

```diff
@@ -163,4 +163,6 @@ default SMSession getWorkspaceSession() {
      */
     void dispose();
 
+    void setRuntimeProperty(String key, String value);
```

Comment:
Please 
- declare getter before setter
- write some comment for getter (what is runtime property)